### PR TITLE
Feature/seab 3698/topic sentence phase2

### PR DESCRIPTION
**Pull request by garyluu**
**Description**
Separate topic into topicManual and topicAutomatic. Also added topic selection.

image

The ul, li structure of the workflow counterpart is kind of a mess due to an existing issue but fixing it now would probably be inefficient when an existing bootstrap issue would probably end up changing it anyways.

**Issue**
SEAB-3698

### CODE REVIEW

**Review by denis-yuen — APPROVED** [review]
How does the screenshot look when there is a long topic for both manual and automatic?

**Comment by garyluu** [comment]
> How does the screenshot look when there is a long topic for both manual and automatic?

Usually can't be done (150 char limit). With the HTML editing magic, it's 
image

File: src/app/search/search-workflow-table/search-workflow-table.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <a [matTooltip]="workflow?.full_workflow_path" [routerLink]="'/workflows/' + workflow?.full_workflow_path">{{
           workflow?.organization + '/' + workflow?.repository + (workflow?.workflowName ? '/' + workflow?.workflowName : '')
         }}</a>
-        <span class="truncate-text-2 size-small" *ngIf="workflow.topic">{{ workflow.topic }}</span>
+        <span class="truncate-text-2 size-small" *ngIf="workflow.topicAutomatic">{{ workflow.topicAutomatic }}</span>
```
Comment (coverbeck) [inline src/app/search/search-workflow-table/search-workflow-table.component.html:20]: Other lines above are using `workflow?`, needed here?

**Review by coverbeck — APPROVED** [review]

File: src/app/search/search-tool-table/search-tool-table.component.html [hunk 2]
```diff
@@ -42,7 +42,7 @@
             tool?.namespace + '/' + tool?.name + (tool?.toolname ? '/' + tool?.toolname : '')
           }}</a>
         </ng-template>
-        <span class="truncate-text-2 size-small" *ngIf="tool.topic">{{ tool.topic }}</span>
+        <span class="truncate-text-2 size-small" *ngIf="tool.topicAutomatic">{{ tool.topicAutomatic }}</span>
```
[the remark this answers is not in the document]
Comment (garyluu) [reply src/app/search/search-tool-table/search-tool-table.component.html:45]: tool.private_access and tool.verified isn't so it's probably not needed but also probably wouldn't hurt to add it

File: src/app/search/search-workflow-table/search-workflow-table.component.html (hunk 1 above)
Comment (garyluu) [reply src/app/search/search-workflow-table/search-workflow-table.component.html:20]: workflow.verified doesn't , same as previous comment.

**Review by NatalieEO — APPROVED** [review]

**Review by wshands — APPROVED** [review]

**Pull request merged**